### PR TITLE
Add "statistics" field to is-valid workers

### PR DESCRIPTION
**Pull request by polinaeterna**
todo: 
- [x]  update openapi.json
- [x] update admin-ui

### CODE REVIEW

File: docs/source/openapi.json [hunk 1]
```diff
@@ -4620,12 +4623,13 @@
                 "examples": {
                   "all the capabilities": {
                     "summary": "valid dataset",
-                    "description": "Try with https://datasets-server.huggingface.co/is-valid?dataset=glue",
+                    "description": "Try with https://datasets-server.huggingface.co/is-valid?dataset=nyu-mll/glue",
```
Comment (polinaeterna) [inline docs/source/openapi.json:4626]: glue is now a community dataset :tada: 
note that we will have to go through all our docs when all canonical datasets will be transferred and update them

Comment (severo) [reply docs/source/openapi.json:4626]: oh right! It's broken on our side: the Hub redirects to the new one, but we don't have the information on the datasets server.

I opened https://github.com/huggingface/datasets-server/issues/2578

File: libs/libcommon/src/libcommon/processing_graph.py [hunk 2]
```diff
@@ -591,7 +591,7 @@ def parse_id(id: str) -> tuple[str, str, Optional[str], Optional[str], str]:
             "split-first-rows",
             "split-duckdb-index",
         ],
-        "job_runner_version": 3,
+        "job_runner_version": 4,
```
[the remark this answers is not in the document]
Comment (severo) [reply libs/libcommon/src/libcommon/processing_graph.py:594]: all the entries will be recomputed the night after the change is deployed

**Review by severo — APPROVED** [review]
Super useful. Once deployed and backfilled, I'll update the Hub

**Pull request merged**